src/Raven.Server/Smuggler/Documents/SmugglerBase.cs

```diff
@@ -981,6 +982,10 @@ await using (var actions = action)
                 {
                     await actions.WriteDatabaseRecordAsync(databaseRecord, result, _options.AuthorizationStatus, _options.OperateOnDatabaseRecordTypes);
                 }
+                catch (LicenseLimitException lle)
+                {
+                    throw new LicenseLimitException(lle.Type, lle.Message);
```

Comment:
It should be:
```
catch (LicenseLimitException)
{
    throw;
}
```

This way you keep the original exception's stackrace

Reply:
Done